examples/stm32/xively_demo_mt/Src/system_stm32f4xx.c

```diff
@@ -1,21 +1,21 @@
 /**
   ******************************************************************************
-  * @file    system_stm32f4xx.c

```

Comment:
I have a solution offer: we can have links to these files inside the SDK. In case only file contents change and no files are removed, added, then it is a cross-SDK solution which prevents such update needs. So it isn't ultimate solution but one step closer.